website/src/components/KnowTheCommunityCarousel/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import KnowTheCommunityCard from '../KnowTheCommunityCard/index';
+import styles from './index.module.css';
+import features from '../../../websiteData/features';
+import { PhCaretLeftFill, PhCaretRightFill } from '../../assets/icon';
+
+const index = () => {
```

Comment:
Please change "index" to a proper component name

Reply:
Sure I will change it. Thanks for commenting that out